build/windows/installer/conf/fluent-bit.conf

```diff
@@ -43,3 +43,16 @@
 #${MultilineEnabled}    Match oms.container.log.la.*
 #${MultilineEnabled}    multiline.key_content log
 #${MultilineEnabled}    multiline.parser #${MultilineLanguages}
+
+#${KubernetesFilterEnabled}[FILTER]
+#${KubernetesFilterEnabled}    Name                kubernetes
+#${KubernetesFilterEnabled}    Match               oms.container.log.la.*
+#${KubernetesFilterEnabled}    Merge_Log           Off
+#${KubernetesFilterEnabled}    Buffer_Size         0
+#${KubernetesFilterEnabled}    Use_Kubelet         true
+#${KubernetesFilterEnabled}    Kubelet_Port        10250
+#${KubernetesFilterEnabled}    Kubelet_Host        ${NODE_IP}
+#${KubernetesFilterEnabled}    tls.verify          Off
+#${KubernetesFilterEnabled}#${AnnotationBasedLogFilteringEnabled}    K8S-Logging.Exclude On
+#${KubernetesFilterEnabled}#${AnnotationBasedLogFilteringEnabled}    Kube_Meta_Cache_TTL 60
```

Comment:
I think, 60 is default good and lets make this configurable through config map if this needs to be smaller value for the highly dynamic workload.